tests/test_config.py

```diff
@@ -0,0 +1,56 @@
+"""Test the Pydantic-based configuration."""
+
+from pathlib import Path
+
+import tomlkit as tkit
+
+import neurogym as ngym
+from neurogym.conf.conf import Conf
+
+
+def test_load_default_config():
+    """Test if the default configuration can be loaded."""
+    ngym.logger.info(f"Logging level: {ngym.conf.monitor.log.level}")
+    ngym.logger.info("Neurogym configuration loaded successfully.")
```

Comment:
is there anything useful to assert here to ensure that the loading happened as expected?

Reply:
Not really...removed.